app/controllers/api/v1/bulk_export_controller.rb

```diff
@@ -17,60 +17,55 @@ def index
         else
           exports = @api_user.bulk_exports
         end
-        render json: exports.to_json(except: [:updated_at, :user_id, :collection_id], include: {:collection => {only: [:title, :slug]}})
+        render json: exports.to_json(except: [:updated_at, :user_id, :collection_id], include: { collection: { only: [:title, :slug] } })
       else
         render status: 401, json: 'You must use an API token to access bulk exports'
       end
     end
 
-
     def start
       if @api_user
         collection_slug = params[:collection_slug]
-        if collection_slug
```

Comment:
This is redundant checking. The route requires /:collection_id so this will always be present. It should suffice that we check if collection.nil? instead.

Reply:
That sounds true, but what are we doing differently with a `collection_slug` parameter versus the usual `collection_id` (which should populate `@collection`?